packages/shared/src/components/comments/CommentActionButtons.tsx

```diff
@@ -167,6 +173,33 @@ export default function CommentActionButtons({
     });
   }
 
+  if (isLoggedIn && comment?.author) {
```

Comment:
Just something that popped up in my head:
1. We should explicit not render if you are the author
2. We should probably exclude 404 user :)

Reply:
The 404 user is generic for the whole follow system actually, maybe we can make 1 generic `fn` that checks if it's you or 404 it shouldn't continue.

Reply:
yes, good thinking